pkg/stores/sqlproxy/proxy_store.go

```diff
@@ -247,7 +282,11 @@ func getFieldForGVK(gvk schema.GroupVersionKind) [][]string {
 }
 
 func keyFromGVK(gvk schema.GroupVersionKind) string {
```

Comment:
Nitpick: as this function is only called once it could be inlined.

Reply:
I missed this first time through. Done now.